x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings/latest_findings_container.tsx

```diff
@@ -95,3 +98,23 @@ const LatestFindingsPageTitle = () => (
     />
   </PageTitle>
 );
+
+const getFindingsDistribution = ({
+  total,
+  passed,
+  failed,
+  currentPageSize,
+  pageIndex,
+  pageSize,
+}: Record<'currentPageSize' | 'total' | 'passed' | 'failed', number | undefined> &
+  Record<'pageIndex' | 'pageSize', number>) => {
```

Comment:
not sure I understand why we accept undefined on some parameters and not on the others. will it still work if `pageIndex` or `pageSize` are `undefined`?